tests/placeAndMakeSpotOrder.ts

```diff
@@ -156,7 +156,10 @@ describe('place and make spot order', () => {
 			marketIndex,
 			direction: PositionDirection.LONG,
 			baseAssetAmount,
-			price: new BN(40).mul(PRICE_PRECISION),
+			price: new BN(41).mul(PRICE_PRECISION),
+			auctionStartPrice: new BN(40).mul(PRICE_PRECISION),
+			auctionEndPrice: new BN(41).mul(PRICE_PRECISION),
+			auctionDuration: 1,
```

Comment:
why was it failing w/o this?

Reply:
the limit order had no auction and it was placed the slot before the maker order, so they couldn't match based on the new rules